src/tests/ftest/server/multiengine_persocket.py

```diff
@@ -159,9 +158,25 @@ def host_reboot(self, hosts):
         """
         reboot_waittime = self.params.get("reboot_waittime", "/run/server_config/*", default=210)
         cmd = "sudo shutdown -r now"
-        run_pcmd(hosts, cmd, timeout=210)
+        run_remote(self.log, hosts, cmd, timeout=210)
         self.log.info("===Server %s rebooting... \n", hosts)
         time.sleep(reboot_waittime)
+        ping_timeout = 600
+        start = time.time()
+        ping_status = None
+        while ping_status is None and (time.time() - start) < ping_timeout:
+            ping_result = run_local(self.log, ["ping", "-c 1", str(hosts)], check=False).stdout
```

Comment:
I'm fairly certain this node designation is not supported for the ping command.  If `hosts`, being a NodeSet object, contains multiple hosts, `str(hosts)` will give you something like `wolf-[118-121]`.  We may need to issue multiple ping commands - one per host - where we could reduce the list of hosts for each host that responds.

Reply:
there are 'fping' and similar commands for doing just this in parallel.